src/main/kotlin/org/jitsi/jicofo/metrics/JicofoMetricsContainer.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Jicofo, the Jitsi Conference Focus.
+ *
+ * Copyright @ 2022 - present 8x8, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jicofo.metrics
+
+import org.jitsi.metrics.MetricsContainer
+
+class JicofoMetricsContainer private constructor() : MetricsContainer() {
+    companion object {
+        @JvmStatic
+        val instance = JicofoMetricsContainer()
```

Comment:
Will Jicofo stats be exposed using the [jicoco-metrics endpoint](https://github.com/jitsi/jicoco/blob/master/jicoco-metrics/src/main/java/org/jitsi/rest/prometheus/Prometheus.java)? If so, this instance should be an instance of `MetricsContainer`, since jicoco-metrics doesn't import jicofo (nor jitsi-videobridge, etc).
See: [VideobridgeMetricsContainer](https://github.com/jitsi/jitsi-videobridge/blob/master/jvb/src/main/kotlin/org/jitsi/videobridge/metrics/VideobridgeMetricsContainer.kt) (not that there's much to see).

Reply:
Yes. But JicofoMetricsContainer and VideobridgeMetricsContainer extend MetricsContainer, so they **are** instances of it.

Reply:
I'll have to go over those commits and see how I got to that point 🤦‍♂️. Let's just call it tunnel vision.